### ISSUE TITLE
Vimeo not working anymore

### ISSUE THREAD

**Issue by Anorax** [rg3/youtube-dl#5644]
Hi there!

Some days ago I was able to download Videos from vimeo.com, but now I always get 404 error messages when I try it.

Output is this:

[debug] System config: []
[debug] User config: []
[debug] Command-line args: [u'-f', u'h264-hd', u'-u', u'PRIVATE', u'-p', u'PRIVATE', u'-4', u'https://vimeo.com/channels/900773/126813964', u'--verbose']
[debug] Encodings: locale UTF-8, fs UTF-8, out UTF-8, pref UTF-8
[debug] youtube-dl version 2015.05.04
[debug] Python version 2.7.6 - Linux-3.13.0-24-generic-x86_64-with-LinuxMint-17-qiana
[debug] exe versions: avconv 9.18-6, avprobe 9.18-6, rtmpdump 2.4
[debug] Proxy map: {}
[vimeo] Logging in
[vimeo] 126813964: Downloading webpage
ERROR: Unable to download webpage: HTTP Error 404: Not Found (caused by HTTPError()); please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
  File "/usr/local/bin/youtube-dl/youtube_dl/extractor/common.py", line 312, in _request_webpage
    return self._downloader.urlopen(url_or_request)
  File "/usr/local/bin/youtube-dl/youtube_dl/YoutubeDL.py", line 1708, in urlopen
    return self._opener.open(req, timeout=self._socket_timeout)
  File "/usr/lib/python2.7/urllib2.py", line 410, in open
    response = meth(req, response)
  File "/usr/lib/python2.7/urllib2.py", line 523, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib/python2.7/urllib2.py", line 448, in error
    return self._call_chain(*args)
  File "/usr/lib/python2.7/urllib2.py", line 382, in _call_chain
    result = func(*args)
  File "/usr/lib/python2.7/urllib2.py", line 531, in http_error_default
    raise HTTPError(req.get_full_url(), code, msg, hdrs, fp)

Is this a youtube-dl problem or a problem on my side?

**Comment by DarKnight--** [comment]
vimeo.com is working perfectly on my system. The link you are trying is a private channel video.

**Comment by rrooij** [comment]
Vimeo works here perfectly too.

**Comment by Anorax** [comment]
Since I have access to this private channel and it worked perfectly till last week... Maybe vimeo changed something on their login API?

**Comment by rrooij** [comment]
@Anorax 
Oh, so it doesn't work on private channels? 

I tried this password protected video and it worked. 

http://vimeo.com/68375962

However, I'm not sure about private channels.

**Comment by Anorax** [comment]
Yep, seems that only private channels are affected. This for example is a paid content channel from a german anime distributor.
Like I said, worked like a charm till last week.

**Comment by machtwebseiten** [comment]
Could this have something to do with the login-token problems at #6488 
Seems related as this also is a login-problem to a video that is protected

**Comment by jubalh** [comment]
Any news on how to solve this?

**Comment by jubalh** [comment]
Or anything one can do to help?
